test/test-ncu.js

```diff
@@ -105,6 +105,16 @@ describe('npm-check-updates', function () {
             childProcess.stdin.end();
         });
 
+        it('should handle no package.json to analyze when receiving empty content on stdin', function (done) {
```

Comment:
Please use the existing `spawn` package for invoking child processes. It provides a clean, promisified interface. Then we won't need to pull in `child_process` to do the same thing. You see how `spawn` is used elsewhere in the file. e.g. `spawn('pwd').then(...)`. Returning a promise from the `it` block automatically handles errors.